hnn_core/__init__.py

```diff
@@ -6,4 +6,4 @@
 from .basket import L2Basket, L5Basket
 from .parallel_backends import MPIBackend, JoblibBackend
 
-__version__ = '0.1'
+__version__ = '0.2.dev0'
```

Comment:
umm ... think you should revert this but make sure the version number in `conf.py` is 0.1 because you want to push this to the 0.1 docs